dyno-core/src/main/java/com/netflix/dyno/connectionpool/impl/HostsUpdater.java

```diff
@@ -15,54 +15,102 @@
  */
 package com.netflix.dyno.connectionpool.impl;
 
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.List;
+import java.util.*;
 import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.concurrent.atomic.AtomicReference;
 
 import com.netflix.dyno.connectionpool.Host;
 import com.netflix.dyno.connectionpool.HostSupplier;
+import com.netflix.dyno.connectionpool.TokenMapSupplier;
+import com.netflix.dyno.connectionpool.exception.DynoException;
 import com.netflix.dyno.connectionpool.exception.NoAvailableHostsException;
+import com.netflix.dyno.connectionpool.impl.lb.HostToken;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class HostsUpdater {
 
-	private final HostSupplier hostSupplier; 
+	private static final org.slf4j.Logger Logger = LoggerFactory.getLogger(ConnectionPoolImpl.class);
```

Comment:
`org.slf4j.Logger` --> `Logger`. You have already imported the package above.

Reply:
done